bool found
bool has_forward
int16[64] ir_forward
bool has_right
int16[64] ir_right